### DISCUSSION TITLE
OG-Core Developer Meeting, Jun. 17, noon-1pm ET

### DISCUSSION THREAD

**Discussion by rickecon** [PSLmodels/OG-Core#942] [Meeting]
**Agenda for OG-Core developer meeting: 6/17/24**
12:00-1:00pm ET

Join Google Meet video conference
Video call link: https://meet.google.com/ebx-znhr-dpb
Or dial: ‪(US) +1 413-298-2642‬ PIN: ‪346 358 145‬#

This meeting is open to the public.

**Agenda**
* Recent OG-Lifespan technical paper
* Open PRs:
    * OG-Core: PR #940 Update automatic versioning
    * OG-Core: PR #941 Add HSV functions to valid tax_func_type
    * OG-USA: [PR #107](https://github.com/PSLmodels/OG-USA/pull/107) Update default calibration
    * OG-ZAF: [PR #54](https://github.com/EAPD-DRB/OG-ZAF/pull/54) Updates compatibility with OG-Core

Next meeting: Monday, Jul. 1, noon-1pm ET

@rickecon @jdebacker @SeaCelo

**Comment by SeaCelo** [comment]
Hi Rick,I can’t join you today but I’ll catch up with you both later this week. Thanks,MarceloOn Jun 17, 2024, at 3:45 AM, Richard Evans ***@***.***> wrote:﻿
Agenda for OG-Core developer meeting: 6/17/24
12:00-1:00pm ET
Join Google Meet video conference
Video call link: https://meet.google.com/ebx-znhr-dpb
Or dial: ‪(US) +1 413-298-2642‬ PIN: ‪346 358 145‬#
This meeting is open to the public.
Agenda

Recent OG-Lifespan technical paper
Open PRs:

OG-Core: PR #940 Update automatic versioning
OG-Core: PR #941 Add HSV functions to valid tax_func_type
OG-USA: PR #107 Update default calibration
OG-ZAF: PR #54 Updates compatibility with OG-Core



Next meeting: Monday, Jul. 1, noon-1pm ET
@rickecon @jdebacker @SeaCelo
